templates/block_creator/block_creator.py

```diff
@@ -1,38 +1,41 @@
+"""Blockcreator template.
+
+used to deploy blockcreator container from autostartable flist.
+"""
+
 import os
 import time
 from random import shuffle
-
+import gevent
 from jumpscale import j
 from zerorobot.service_collection import ServiceNotFoundError
 from zerorobot.template.base import TemplateBase
 from zerorobot.template.decorator import retry
 from zerorobot.template.state import StateCheckError
-
-CONTAINER_TEMPLATE_UID = 'github.com/threefoldtech/0-templates/container/0.0.1'
-PEER_DISCOVERY_TEMPLATE_UID = 'github.com/threefoldtoken/0-templates/peer_discovery/0.0.1'
+import ipaddress
 
 
 class BlockCreator(TemplateBase):
-    version = '0.0.1'
+    """Blockcreator template used to deploy blockcreator tfchaind on a
+    container using 0-robot."""
+
+    version = '0.0.2'
     template_name = 'block_creator'
 
     _DATA_DIR = '/mnt/data'
     _BACKUP_DIR = '/mnt/backups'
 
     def __init__(self, name=None, guid=None, data=None):
+        """Initialize blockcreator template."""
         super().__init__(name=name, guid=guid, data=data)
         # bind uninstall action to the delete method
         self.add_delete_callback(self.uninstall)
         self._node_sal = j.clients.zos.get('local')
+        self.__client_sal = None
 
-        wallet_passphrase = self.data.get('walletPassphrase')
-        if not wallet_passphrase:
-            self.data['walletPassphrase'] = j.data.idgenerator.generateGUID()
-
-        # peer discovery service
-        self._discovery = None
-
-        self.recurring_action('_monitor', 30)  # every 30 seconds
+        # wallet_passphrase = self.data.get('walletPassphrase')
```

Comment:
So now we don't support encrypted wallet anymore at all ?

Reply:
yup, according to this comment  https://github.com/threefoldtech/home/issues/79#issuecomment-456337514

Reply:
remove the commented code then